pkgs/by-name/xe/xeus/package.nix

```
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
   pname = "xeus";
-  version = "3.2.0";
+  version = "5.1.1";
 
   src = fetchFromGitHub {
     owner = "jupyter-xeus";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-D/dJ0SHxTHJw63gHD6FRZS7O2TVZ0voIv2mQASEjLA8=";
+    repo = "xeus";
+    rev = "5.1.1";
```

```suggestion
    tag = version;
```
It's common to have the `rev` or `tag` argument be `version`, `rg --stats 'tag = version;'` finds 1971 matches.

---

Thanks. Also, changing  repo to `pname`:
```
    repo = pname;
    tag = version;
```